README.md

```diff
@@ -215,7 +217,7 @@ Open a terminal and start darktable with the command `darktable -d lua`. This pr
 
 ### Windows
 
-Open the Git Bash Shell. Start darktable with the command `/c/Program\ Files/darktable/bin/darktable -d lua`. This provides debugging information to give you insight into what is happening.
+Open a command prompt. Start darktable with the command C:\"Program Files\darktable\bind\darktable" -d lua > log.txt. This provides debugging information to give you insight into what is happening.
```

Comment:
`C:\"Program Files\darktable\bind\darktable`
Are you shure ...\\bind\\... is right?

Reply:
I think my fingers stuttered when I was typing and then I saw what I expected to see.  Thanks for the catch